TASK-53005: In Tasks application, updating name project results loss of project.

IlhemEssaadi opened the pull request:
ISSUES : In Tasks application, when updating the project name, the updates are not immediately made and the display of the project is lost after refreshing.
FIX : the problem is in the condition of reset method which is launched when we refresh the page and it fixed by fixing the condition and the second problem is that the manager and participator variables get the wrong value, so it is fixed by allow manager and participator variables to get the value of GroupSpace.

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

hthabti committed:
TASK-51872: can't remove start date of a task (#728)

issue: start date can't be removed when clicking on 'none' label from the datepicker
fix: add else-if statement to handle user's input on the datepicker when the value of entered label = 'none'

Amine0008 committed:
removed the blur event (#743)

IlhemEssaadi committed:
TASK-53005: In Tasks application, when updating the project name, the updates are not immediately made and the display of the project is lost after updating.

ISSUES : In Tasks application, when updating the project name, the updates are not immediately made and the display of the project is lost after updating.
FIX : the problem is in the reset method which is launched when we refresh the page and it fixed by fixing the condition .

hthabti committed:
TASK-51872: Cannot remove start date of a task (#741)

* TASK-51872: can't remove start date of a task (#739)

rdenarie reviewed (changes requested):
This PR revert a change done few day ago.

hthabti committed:
TASK-51872: Cannot remove start date of a task (#746)

issue: start date can't be removed when clicking on 'none' label from the datepicker
fix: add else-if statement to handle user's input on the datepicker when the value of entered label = 'none'

IlhemEssaadi committed:
update that allows manager and participator  to get the value of Group space

hbenali commented:
You better shorten the pull request title, it's too long

rdenarie committed:
Merge branch 'feature/maintenance' into TASK-53005

rdenarie reviewed (changes requested):
Can you shorter the PR title ?

In addition can you explain in the description how you fix the problem ? What is the modification. ? currently you explain only where is the problem but not what it is

rdenarie committed:
Merge branch 'feature/maintenance' into TASK-53005
